.ci/appveyor.yml

```diff
@@ -56,6 +56,15 @@ install:
   - "npm config set loglevel warn"
   - "npm install"
 
+  - curl -fsSL ftp://ftp.zlatkovic.com/libxml/iconv-1.9.2.win32.zip -o iconv.zip
```

Comment:
Please don't user curl. It caused problems in another place: https://github.com/coala/coala/pull/3962. Use the builtin ``DownloadFile``